jmix-email/email-flowui/src/main/resources/io/jmix/emailflowui/view/sendingmessage/sending-message-view.xml

```diff
@@ -50,9 +50,6 @@
     </data>
     <facets>
         <dataLoadCoordinator auto="true"/>
-        <settings>
```

Comment:
it's better to keep settings facet, otherwise changing column sizes and collapsing them won't persist